autoload.php

```diff
@@ -0,0 +1,31 @@
+<?php
+namespace Onedrive;
+
+// The Onedrive SDK autoloader.  You probably shouldn't be using this.  Instead,
+// use a global autoloader, like the Composer autoloader.
+//
+// But if you really don't want to use a global autoloader, do this:
+//
+//     require_once "<path-to-here>/Onedrive/autoload.php"
```

Comment:
This path currently seems incorrect, it should be `<path-to-here>/autoload.php`, but we should probably just say `__DIR__ . '/autoload.php'` as the `<path-to-here>` is confusing (path to this file? or path to the containing directory?).